fvm/derived/derived_chain_data_test.go

```diff
@@ -19,7 +19,7 @@ func TestDerivedChainData(t *testing.T) {
 
 	testLocation := func(hex string) common.AddressLocation {
 		return common.AddressLocation{
-			Address: common.Address(flow.HexToAddress(hex)),
+			Address: common.MustBytesToAddress(flow.HexToAddress(hex).Bytes()),
```

Comment:
We used to do it this style, but there were some issues with addresses starts with zero bytes, 
Maybe just a final approval by @turbolent and @janezpodhostnik that is safe to use this now.

Reply:
There are some inconsistency in how cadence handle address (In particular, cadence.Address and cadence/runtime/common.Address have different behaviour.  cadence.Address's BytesToAddress doesn't truncate incorrectly length address correctly).

it's safe in this case since we're using common.Address.